### ISSUE TITLE
missing permissions for several files

### ISSUE THREAD

**Issue by mmuehlbeyer** [HermesGermany/galapagos#261]
Hi,

after downloading and testing the repo I've discovered missing execution privileges for 
all of the bash files. 
e.g start-demo.sh, everything in demo-cloud

```
ls demo-ccloud/*.sh
demo-ccloud/install-keycloak.sh demo-ccloud/install-yq.sh       demo-ccloud/setup-demo.sh
```
is that intended?

also after downloading  yq its missing execute rights:

```
Downloading yq for darwin_arm64...
[...]
./setup-demo.sh: line 98: ./bin/yq: Permission denied
```

**Comment by albrechtflo-hg** [comment]
Hi @mmuehlbeyer , thanks for the valid bug report. We are planning to fix it in the next release.

**Comment by albrechtflo-hg** [comment]
Fixed via #262
